modules/installation-azure-config-yaml.adoc

```diff
@@ -175,93 +224,138 @@ endif::openshift-origin[]
 endif::gov[]
 ----
 ifndef::gov[]
-<1> Required. The installation program prompts you for this value.
+<1> Required. The `baseDomain` parameter specifies the base domain of your cloud provider. The installation program prompts you for this value.
 endif::gov[]
 ifdef::gov[]
 <1> Required.
 endif::gov[]
+ifndef::azure-restricted[]
 <2> If you do not provide these parameters and values, the installation program provides the default value.
+endif::azure-restricted[]
+ifdef::azure-restricted[]
+<2> The configuration for the machines that comprise the control plane. The `controlPlane` section is a single mapping. The first line of the `controlPlane` section must not begin with a hyphen, `-`. You can use only one control plane pool. If you do not provide these parameters and values, the installation program provides the default value.
+endif::azure-restricted[]
+ifndef::azure-restricted[]
```

Comment:
No need to add these lines. We can reuse the description of  <2>.

Reply:
+1